toolchain/check/type_completion.cpp

```diff
@@ -616,18 +616,16 @@ auto RequireIdentifiedFacetType(Context& context,
       context.facet_types().Get(facet_type.facet_type_id);
 
   SemIR::IdentifiedFacetType result;
-  result.required_interfaces = facet_type_info.impls_constraints;
+  result.mutable_required_interfaces() = facet_type_info.impls_constraints;
```

Comment:
would `set_required_interfaces(facet_type_info.impls_constraints)` work?

Reply:
Done. I may have to revisit this once we add support for named constraints, but this simpler API will work until then.